src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL;
+    private static final Properties DB_PROPERTIES;
```

Comment:
there is no constant: 
constant should be unchangeable through all workflow (e.g. Immutable)
```suggestion
    private static final Properties DB_PROPERTIES;
```